manifests/agent.pp

```diff
@@ -264,8 +264,9 @@
   # can find the ipaddress of this specific interface if listenip
   # is set to for example "eth1" or "bond0.73".
   if ($listenip != undef) {
-    if ($listenip =~ /^(eth|bond|lxc|eno|tap|tun).*/) {
+    if ($listenip =~ /^(eth|lo|bond|lxc|eno|tap|tun).*/) {
       $int_name  = getvar("::ipaddress_${listenip}")
+      $listen_ip = $int_name
```

Comment:
Are you suggesting not using $int_name, and replacing it with:
`$listen_ip = getvar("::ipaddress_${listenip}")` ? If $int_name is not used, then go for it. I didn't want to break the module by removing a variable I didn't understand.